[debug] [2019-05-21T10:11:20.511Z] ----------------------------------------------------------------------
[debug] [2019-05-21T10:11:20.514Z] Command:       /usr/local/Cellar/node/11.4.0/bin/node /usr/local/bin/firebase serve --only functions
[debug] [2019-05-21T10:11:20.514Z] CLI Version:   6.10.0
[debug] [2019-05-21T10:11:20.514Z] Platform:      darwin
[debug] [2019-05-21T10:11:20.514Z] Node Version:  v10.15.3
[debug] [2019-05-21T10:11:20.516Z] Time:          Tue May 21 2019 19:11:20 GMT+0900 (GMT+09:00)
[debug] [2019-05-21T10:11:20.516Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-05-21T10:11:20.526Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-05-21T10:11:20.526Z] > authorizing via signed-in user
[debug] [2019-05-21T10:11:20.526Z] [iam] checking project titan-dev-1234 for permissions ["firebase.projects.get"]
[debug] [2019-05-21T10:11:20.529Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/titan-dev-1234:testIamPermissions  

[debug] [2019-05-21T10:11:25.982Z] <<< HTTP RESPONSE 200
[debug] [2019-05-21T10:11:25.985Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/titan-dev-1234  
 
[debug] [2019-05-21T10:11:27.766Z] <<< HTTP RESPONSE 200
[debug] [2019-05-21T10:11:28.178Z] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:125:11)
    at Object.join (path.js:1147:7)
    at new FunctionsEmulator (/usr/local/lib/node_modules/firebase-tools/lib/emulator/functionsEmulator.js:44:34)
    at Object.<anonymous> (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:36:71)
    at Generator.next (<anonymous>)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:3:12)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:16:16)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:23
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-05-21T10:11:50.139Z] ----------------------------------------------------------------------
[debug] [2019-05-21T10:11:50.141Z] Command:       /usr/local/Cellar/node/11.4.0/bin/node /usr/local/bin/firebase serve --only functions
[debug] [2019-05-21T10:11:50.141Z] CLI Version:   6.10.0
[debug] [2019-05-21T10:11:50.141Z] Platform:      darwin
[debug] [2019-05-21T10:11:50.141Z] Node Version:  v10.15.3
[debug] [2019-05-21T10:11:50.142Z] Time:          Tue May 21 2019 19:11:50 GMT+0900 (GMT+09:00)
[debug] [2019-05-21T10:11:50.142Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-05-21T10:11:50.154Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-05-21T10:11:50.154Z] > authorizing via signed-in user
[debug] [2019-05-21T10:11:50.155Z] [iam] checking project titan-dev-1234 for permissions ["firebase.projects.get"]
[debug] [2019-05-21T10:11:50.156Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/titan-dev-1234:testIamPermissions  

[debug] [2019-05-21T10:11:51.368Z] <<< HTTP RESPONSE 200
[debug] [2019-05-21T10:11:51.370Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/titan-dev-1234  
 
[debug] [2019-05-21T10:11:52.870Z] <<< HTTP RESPONSE 200
[debug] [2019-05-21T10:11:54.059Z] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:125:11)
    at Object.join (path.js:1147:7)
    at new FunctionsEmulator (/usr/local/lib/node_modules/firebase-tools/lib/emulator/functionsEmulator.js:44:34)
    at Object.<anonymous> (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:36:71)
    at Generator.next (<anonymous>)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:3:12)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:16:16)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:23
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-05-21T10:17:17.470Z] ----------------------------------------------------------------------
[debug] [2019-05-21T10:17:17.473Z] Command:       /usr/local/Cellar/node/11.4.0/bin/node /usr/local/bin/firebase serve --only functions
[debug] [2019-05-21T10:17:17.473Z] CLI Version:   6.10.0
[debug] [2019-05-21T10:17:17.473Z] Platform:      darwin
[debug] [2019-05-21T10:17:17.473Z] Node Version:  v10.15.3
[debug] [2019-05-21T10:17:17.478Z] Time:          Tue May 21 2019 19:17:17 GMT+0900 (GMT+09:00)
[debug] [2019-05-21T10:17:17.479Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-05-21T10:17:17.491Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-05-21T10:17:17.491Z] > authorizing via signed-in user
[debug] [2019-05-21T10:17:17.492Z] [iam] checking project titan-dev-1234 for permissions ["firebase.projects.get"]
[debug] [2019-05-21T10:17:17.493Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/titan-dev-1234:testIamPermissions  

[debug] [2019-05-21T10:17:19.708Z] <<< HTTP RESPONSE 200
[debug] [2019-05-21T10:17:19.710Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/titan-dev-1234  
 
[debug] [2019-05-21T10:17:21.163Z] <<< HTTP RESPONSE 200
[debug] [2019-05-21T10:17:22.061Z] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:125:11)
    at Object.join (path.js:1147:7)
    at new FunctionsEmulator (/usr/local/lib/node_modules/firebase-tools/lib/emulator/functionsEmulator.js:44:34)
    at Object.<anonymous> (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:36:71)
    at Generator.next (<anonymous>)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:3:12)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:16:16)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:23
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-05-21T10:22:12.099Z] ----------------------------------------------------------------------
[debug] [2019-05-21T10:22:12.102Z] Command:       /usr/local/Cellar/node/11.4.0/bin/node /usr/local/bin/firebase serve --only functions
[debug] [2019-05-21T10:22:12.102Z] CLI Version:   6.10.0
[debug] [2019-05-21T10:22:12.102Z] Platform:      darwin
[debug] [2019-05-21T10:22:12.102Z] Node Version:  v10.15.3
[debug] [2019-05-21T10:22:12.107Z] Time:          Tue May 21 2019 19:22:12 GMT+0900 (GMT+09:00)
[debug] [2019-05-21T10:22:12.107Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-05-21T10:22:12.118Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-05-21T10:22:12.118Z] > authorizing via signed-in user
[debug] [2019-05-21T10:22:12.119Z] [iam] checking project titan-dev-1234 for permissions ["firebase.projects.get"]
[debug] [2019-05-21T10:22:12.120Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/titan-dev-1234:testIamPermissions  

[debug] [2019-05-21T10:22:14.260Z] <<< HTTP RESPONSE 200
[debug] [2019-05-21T10:22:14.263Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/titan-dev-1234  
 
[debug] [2019-05-21T10:22:16.286Z] <<< HTTP RESPONSE 200
[debug] [2019-05-21T10:22:17.363Z] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:125:11)
    at Object.join (path.js:1147:7)
    at new FunctionsEmulator (/usr/local/lib/node_modules/firebase-tools/lib/emulator/functionsEmulator.js:44:34)
    at Object.<anonymous> (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:36:71)
    at Generator.next (<anonymous>)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:3:12)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:16:16)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:23
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-05-21T10:22:58.975Z] ----------------------------------------------------------------------
[debug] [2019-05-21T10:22:58.977Z] Command:       /usr/local/Cellar/node/11.4.0/bin/node /usr/local/bin/firebase serve --only functions
[debug] [2019-05-21T10:22:58.978Z] CLI Version:   6.10.0
[debug] [2019-05-21T10:22:58.978Z] Platform:      darwin
[debug] [2019-05-21T10:22:58.978Z] Node Version:  v10.15.3
[debug] [2019-05-21T10:22:58.985Z] Time:          Tue May 21 2019 19:22:58 GMT+0900 (GMT+09:00)
[debug] [2019-05-21T10:22:58.985Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-05-21T10:22:58.996Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-05-21T10:22:58.997Z] > authorizing via signed-in user
[debug] [2019-05-21T10:22:58.997Z] [iam] checking project titan-dev-1234 for permissions ["firebase.projects.get"]
[debug] [2019-05-21T10:22:58.999Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/titan-dev-1234:testIamPermissions  

[debug] [2019-05-21T10:23:00.432Z] <<< HTTP RESPONSE 200
[debug] [2019-05-21T10:23:00.434Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/titan-dev-1234  
 
[debug] [2019-05-21T10:23:01.658Z] <<< HTTP RESPONSE 200
[debug] [2019-05-21T10:23:03.519Z] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:125:11)
    at Object.join (path.js:1147:7)
    at new FunctionsEmulator (/usr/local/lib/node_modules/firebase-tools/lib/emulator/functionsEmulator.js:44:34)
    at Object.<anonymous> (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:36:71)
    at Generator.next (<anonymous>)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:3:12)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:16:16)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:23
[error] 
[error] Error: An unexpected error has occurred.
